proposed/extended-coding-style-guide.md

```diff
@@ -75,7 +75,7 @@ class Foo extends Bar implements FooInterface
 Code MUST follow all rules outlined in [PSR-1].
 
 The term 'StudlyCaps' in PSR-1 MUST be interpreted as PascalCase where the first letter of
-each word is capitalised including the very first letter.
```

Comment:
I feel I should point out that "capitalised" is the correct English spelling and "capitalized" is the Americanised version. It may be a trivial point; nevertheless, I thought I would bring it to your attention in case there was an existing convention with existing PSRs.

Reply:
According to [Oxford](https://en.oxforddictionaries.com/definition/capitalize) capitalized is the correct English spelling.